cmd/cmd_linux.go

```diff
@@ -144,6 +144,12 @@ func InternalToolPath(tool string) (string, error) {
 			prefix := exe[:idx]
 			return filepath.Join(prefix, "/usr/lib/snapd", tool), nil
 		}
+		// or perhaps some other random location, make sure the tool
```

Comment:
I would be slightly happier if we did this only if idx == -1

Reply:
Updated.